test/e2e/storage/testsuites/testdriver.go

```diff
@@ -201,6 +204,26 @@ type StressTestOptions struct {
 	NumRestarts int
 }
 
+// Metrics to evaluate performance of an operation
+// TODO: Add metrics like median, mode, standard deviation, percentile
+type Metrics struct {
+	Latency    time.Duration
+	Throughput float64
+}
+
+// PerformanceTestProvisioningOptions contains parameters for
+// testing provisioning operation performance
+type PerformanceTestProvisioningOptions struct {
```

Comment:
Can we enable these options for hostpath and mock csi drivers?

Reply:
What values do we default to? Looking at `StressTestOptions`, its enabled for `gcePDCSIDriver` with some default values and also configurable?